src/assertions.ts

```diff
@@ -1296,6 +1305,86 @@ function containsJSON(str: string): any {
   return jsonObjects;
 }
 
+export async function isSql(
+  outputString: string,
+  renderedValue: AssertionValue | undefined,
+  inverse: boolean,
+  assertion: Assertion,
+): Promise<GradingResult> {
+  let pass: boolean = false;
```

Comment:
nit: `boolean` type is implied 
```suggestion
  let pass = false;
```

Reply:
got it